account_payment_credit_card/models/account_move.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017 Open Source Integrators <http://www.opensourceintegrators.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
```

Comment:
Please remove empty line, between header and first import only 1 empty line AFAIR